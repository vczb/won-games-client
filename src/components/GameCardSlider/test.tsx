import { render, screen } from '@testing-library/react'

import GameCardSlider from '.'

describe('<GameCardSlider />', () => {
  it('should render the reading', () => {
    // const { container } = render(<GameCardSlider />)
    // expect(
    //   screen.getByRole('heading', { name: /GameCardSlider/i })
    // ).toBeInTheDocument()
    // expect(container.firstChild).toMatchSnapshot()
  })
})
