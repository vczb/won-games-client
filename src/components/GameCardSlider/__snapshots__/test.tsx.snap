// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GameCardSlider /> should render the reading 1`] = `
<div
  class=""
>
  <h1>
    GameCardSlider
  </h1>
</div>
`;
